src/xorable.rs

```diff
@@ -0,0 +1,436 @@
+// Copyright 2018 MaidSafe.net limited.
```

Comment:
This license header should be updated to the BSD/MIT header (can copy header from lib.rs)